tests/Load/config.sh

```
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+prompt_for_input() {
+  local prompt_message=$1
+  local default_value=$2
+  local user_input
+
+  read -p "$prompt_message [$default_value]: " user_input
```

Shellcheck

[info] read without -r will mangle backslashes.

(SC2162)